app/code/community/Oyst/OneClick/Model/Catalog.php

```diff
@@ -507,6 +517,23 @@ protected function _addAmount(Mage_Catalog_Model_Product $product, OystProduct &
      */
     protected function _getPrices(Mage_Catalog_Model_Product $product, $storeId = null)
     {
+        $customPrice = 0;
+        if ($this->configurableProductCustomOptions) {
+            foreach (Zend_Json::decode($this->configurableProductCustomOptions) as $option) {
+                preg_match('!\d+!', $option->optionId, $matches);
+                $customProduct = Mage::getModel('catalog/product')->load($product->getId());
+
+                // @codingStandardsIgnoreLine
```

Comment:
```// @codingStandardsIgnoreLine```
have to be add before 
```$customProduct = Mage::getModel('catalog/product')->load($product->getId());```

![image](https://user-images.githubusercontent.com/861701/35278813-6e1c3b1e-004b-11e8-8601-f1f0a40e88f5.png)